src/components/articleDescription/Index.jsx

```diff
@@ -12,6 +13,6 @@ const ArticleDescription = (props) => {
 };
 
 ArticleDescription.propTypes = {
-  description: PropTypes.string.isRequired
+  // description: PropTypes.string.isRequired
```

Comment:
@Dubby20 please remove this if not in use

Reply:
Alright. It was a mistake